packages/core/src/App/Containers/AccountSwitcherWallet/__tests__/account-switcher-wallet-item.spec.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import { TStores } from '@deriv/stores/types';
+import { AccountSwitcherWalletItem } from '../account-switcher-wallet-item';
+
+jest.mock('@deriv/hooks', () => ({
+    ...jest.requireActual('@deriv/hooks'),
+    useActiveAccount: jest.fn(() => ({ data: { loginid: 'CR007' } })),
+}));
+
+const account: React.ComponentProps<typeof AccountSwitcherWalletItem>['account'] = {
+    currency: 'USD',
+    // @ts-expect-error partial currency_config
+    currency_config: { type: 'fiat', name: 'US dollar', display_code: 'USD' },
+    dtrade_balance: 100,
+    dtrade_loginid: 'CR007',
+    gradients: {
+        card: { dark: 'wallet-card__usd-bg--dark', light: 'wallet-card__usd-bg' },
+        header: { dark: 'wallet-header__usd-bg--dark', light: 'wallet-header__usd-bg' },
+    },
+    icons: { dark: 'icons__dark', light: 'icons__light' },
+    is_active: true,
+    is_virtual: false,
+    landing_company_name: 'svg',
+    linked_to: [{ loginid: 'CR007' }],
+};
+
+const props: React.ComponentProps<typeof AccountSwitcherWalletItem> = {
+    closeAccountsDialog: jest.fn(),
+    account,
+};
+
+const AccountSwitcherWalletItemComponent = ({
+    props,
+    store,
+}: {
+    props: React.ComponentProps<typeof AccountSwitcherWalletItem>;
+    store: TStores;
```

Comment:
```suggestion
    store: ReturnType<typeof mockStore>;
```
just suggestion you can ignore it